src/backends/coreaudio.rs

```diff
@@ -374,3 +479,33 @@ impl<Callback: 'static + Send + AudioOutputCallback> CoreAudioStream<Callback> {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    #[ignore = "This is an integration test that requires a default audio device on macOS."]
```

Comment:
```suggestion
```

you're already checking in the test, the test is going to pass without a default device with your implementation.